CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ tvm_option(USE_OPENGL "Build with OpenGL" OFF)
 tvm_option(USE_METAL "Build with Metal" OFF)
 tvm_option(USE_ROCM "Build with ROCM" OFF)
 tvm_option(ROCM_PATH "The path to rocm" /opt/rocm)
+tvm_option(USE_HEXAGON "Build with Hexagon" OFF)
+tvm_option(USE_HEXAGON_SDK "Path to the Hexagon SDK root (required for Hexagon support)" /path/to/sdk)
```

Comment:
consider combine with USE_HEXAGON, see implementation of USE_LLVM option

Reply:
I think it maybe a little different with LLVM. Because it will distinguish simulator of DSP or real hardware.

Reply:
@FrozenGene Could you elaborate why we cannot simply use one `USE_HEXAGON`? Thanks!

Reply:
in `Hexagon.cmake`, the author has different logic for simulator / real hardware, this depends on `USE_HEXAGON == sim or device`. Need the author to clarify whether we could combine them.

Reply:
@FrozenGene Gotcha. Thanks!

Reply:
@FrozenGene is correct here: we support two compilation modes, one for use with the Hexagon simulator, the other one for use with the actual hardware.  We use USE_HEXAGON setting to distinguish between these two, but we still need an extra parameter to specify the path to the Hexagon SDK.

Reply:
I find `device` target just contain more include header directory. Maybe we could combine them. @kparzysz Could you help to clarify it ?

Reply:
The header in fastrpc directory is an output from the IDL compilation.  It's separated from the rest because it's special in that way.  It should not be edited in place (except for formatting purposes).